holidays/countries/india.py

```diff
@@ -227,16 +227,16 @@ def _populate_public_holidays(self):
 
         # Christian holidays.
 
-        # Palm Sunday.
+        # # Palm Sunday.
         self._add_palm_sunday(tr("Palm Sunday"))
 
-        # Good Friday.
+        # # Good Friday.
         self._add_good_friday(tr("Good Friday"))
 
-        # Easter Sunday.
+        # # Easter Sunday.
         self._add_easter_sunday(tr("Easter Sunday"))
 
-        # Christmas Day.
+        # # Christmas Day.
         self._add_christmas_day(tr("Christmas Day"))
```

Comment:
You might have accidentally put an extra `# ` here for these 4 entries

Reply:
NVM, this seems to be fixed in newer commits - for some reason I saw older ones for a sec